src/saturn_engine/worker_manager/config/declarative_dynamic_topology.py

```diff
@@ -0,0 +1,23 @@
+import typing as t
+
+import dataclasses
+
+from saturn_engine.utils import inspect as extra_inspect
+from saturn_engine.utils.declarative_config import BaseObject
+from saturn_engine.worker_manager.config.static_definitions import StaticDefinitions
+
+
+@dataclasses.dataclass
+class DynamicTopologySpec:
+    module: str
+
+
+@dataclasses.dataclass
+class DynamicTopology(BaseObject):
+    spec: DynamicTopologySpec
+
+    def to_static_definitions(self) -> StaticDefinitions:
+        return t.cast(
```

Comment:
wrap in try/except? This sounds very annoying to crash the whole things to one function.

Reply:
Maybe in `src/saturn_engine/worker_manager/config/declarative.py` actually (the try/except)